ci_framework/roles/hive/tasks/oc-apply.yml

```diff
@@ -0,0 +1,38 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+# Renders the template and then applies the resource.
+#
+# Required variables are
+#
+#  desc            -> description of the task
+#  tempalteFile    -> the resource template to be rendered
+#  workingDir      -> Location to use for storing run artifacts
+#  destFile        -> the name of file to be used after rendation
+#  oc_cli	   -> absolute path to OC CLI
+-
+  name: Rendering the {{ desc }}
+  ansible.builtin.template:
+    src: "{{ templateFile }}"
+    dest: "{{ workingDir }}/{{ destFile }}"
```

Comment:
Going with the above thought process of using `artifacts` could be change it to `cifmw_hive_artifacts_dir`